metaswitch/common/alarms.py

```diff
@@ -46,6 +46,10 @@
 sendrequest = None
 
 def issue_alarm(process, identifier):
+    # Clearwater nodes all have clearwater-infrastructure installed.
+    # It includes a command-line script that can be used to issue an alarm.
```

Comment:
I might add a link to https://github.com/Metaswitch/clearwater-infrastructure/blob/master/clearwater-infrastructure/usr/share/clearwater/bin/alarms.py